poll/public/handlebars/poll_results.handlebars

```diff
@@ -0,0 +1,43 @@
+<script id="poll-results-template" type="text/html">
+    <h3 class="poll-header">{{display_name}}</h3>
+    <div class="poll-question-container">{{{question}}}</div>
+    <ul class="poll-answers-results poll-results">
+    {{#each tally}}
+        <li class="poll-result">
+            <div class="poll-result-input-container">
+              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
+            </div>
+            {{#if any_img}}
+                <div class="poll-image result-image">
+                    <label for="answer-{{key}}" class="poll-image-label">
+                        {{#if img}}
+                          <img src="{{img}}" />
+                        {{/if}}
+                    </label>
+                </div>
+            {{/if}}
+            <div class="percentage-gauge-container">
+              <div class="percentage-gauge" style="width:{{percent}}%;">
+                  <label class="poll-answer-label" for="answer-{{key}}">{{{answer}}}</label>
+              </div>
+            </div>
+            <div class="poll-percent-container">
+                <span class="poll-percent-display{{#if first}} poll-top-choice{{/if}}">{{percent}}%</span>
+            </div>
+        </li>
+       {{^last}}
+        <li class="poll-spacer">
```

Comment:
@cptvitamin This was the second thing (of many) I tried, but unfortunately it works on neither Chrome nor Firefox.  (Just curious: what browser did this you get this to work on?)

If this is pretty bad for screen readers, we have to find a different solution.

@antoviaque Do you have any suggestions how to proceed?  Simply removing the spacers here will change the style for McKinsey, which we probably don't want to do, and recreating the same layout we currently have on all browsers using an approach that doesn't need the spacers is a lot of work and likely to introduce styling incompatibilities and bugs.